tests/test_sidebar_prototype.py

```
@@ -123,29 +234,98 @@ def test_the_drawer_has_a_narrow_screen_rule():
+    body = re.sub(r"/\*.*?\*/", "", css, flags=re.S)
+    selectors = [part.strip()
+                 for block in re.findall(r"([^{}]+)\{", body)
+                 for part in block.split(",")
+                 if part.strip() and not part.strip().startswith("@")]
+
+    # Assert
+    unscoped = [s for s in selectors if "proto-" not in s]
+    assert not unscoped, f"these rules would restyle the live page: {unscoped}"
+
+
+def _client():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_client`

(ANN202)